start_dbt.sh

```
@@ -24,65 +56,157 @@ if [ "$gpg_format" = "ssh" ] && [ -n "$signing_key" ] && [ "$auto_sign" = "true"
+# 5. Python venv for the helper scripts in scripts/ (optional - not needed for dbt)
+# ---------------------------------------------------------------------------
+echo "Syncing Python tooling for scripts/..."
+if ! command -v uv &> /dev/null; then
+    echo "[INFO] uv not found - installing..."
+    curl -LsSf https://astral.sh/uv/install.sh | sh || echo "[WARNING] uv install failed"
+    export PATH="$HOME/.local/bin:$PATH"
+fi
 if command -v uv &> /dev/null; then
     uv sync
-    echo "[OK] Dependencies up to date"
+    [ -f ".venv/bin/activate" ] && source .venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: .venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)